plugins/modules/monitoring/datadog/datadog_monitor.py

```diff
@@ -228,6 +228,7 @@ def main():
             name=dict(required=True),
             query=dict(),
             notification_message=dict(no_log=True),
+            message=dict(no_log=True),
```

Comment:
`message` is internally used by the Ansible Core engine for sys logging. Please use a different parameter name.